ansible/group_vars/lab/lab.yml

```diff
@@ -29,6 +29,10 @@ tacacs_ro_user_passwd: '123456'
 tacacs_jit_user: test_jituser
 tacacs_jit_user_passwd: '123456'
 tacacs_jit_user_membership: netuser
+tacacs_authorization_user: test_auuser
+tacacs_authorization_user_passwd: '123456'
+tacacs_local_user: test_louser
```

Comment:
>tacacs_local_user [](http://example.com/codeflow?start=0&length=17)

Does local_user has anything to do with tacacs? If not, just call it `local_user` ?

Reply:
Fixed.